tests/vppoutputencode.cpp

```diff
@@ -126,6 +126,11 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 }
 
+VppOutputEncode::VppOutputEncode()
+{
+    m_outputBuffer.format = OUTPUT_BUFFER_LAST;
```

Comment:
m_outputBuffer.format has a initial value before this patch when --disable-capi
`#ifndef __ENABLE_CAPI__		
      VideoEncOutputBuffer():data(0), bufferSize(0), dataSize(0)		
     , remainingSize(0), flag(0),  **format(OUTPUT_BUFFER_LAST)**, timeStamp(0) {		
     };		
 #endif`

It's okay to delete this piece of code. I had tested this patch without this.